src/bunnicula/component/connection.clj

```diff
@@ -55,6 +64,7 @@
      :port port
      :username username
      :password password
+     :secure? secure?
```

Comment:
if we get the host from the URL, shouldn't the secure come from the scheme? 
Like `:secure? (= (.getScheme uri) "ampqs")` ?

Reply:
Yes, that's a good point. I could also rewrite the code to just... not do anything - this work is unnecessary if we get a connection URL because in the end - we turn the connection map back to an URI: https://github.com/nomnom-insights/nomnom.bunnicula/blob/9c06e9af0a27d9a64e3dc8361814d82c026d2020/src/bunnicula/component/connection.clj#L35
The only reason why we might keep this code is better logging and automatically using the username for connection name (it's sometimes handy, although Rabbit's connection metrics provide us with both username and connection name ¯\_(ツ)_/¯)

Maybe worth holding with this one until V3 arrives (#3) .